balancer-js/src/modules/sdk.helpers.ts

```diff
@@ -24,3 +40,50 @@ export function getNetworkConfig(
     },
   };
 }
+
+export function getDataRepositories(
```

Comment:
all data sources can be exposed as a data module, for direct usage

comment on the sdk.module